api/envoy/api/v2/route/route.proto

```diff
@@ -585,6 +585,18 @@ message RouteAction {
       // Connection properties hash policy.
       ConnectionProperties connection_properties = 3;
     }
+    // Reserve for new policy_specifier.
+    /* RESERVED */ reserved 4 to 20;
```

Comment:
Out of curiosity, why are we reserving these field numbers?

Reply:
I think we will add more hash method into the oneof list above and I naively think it'd be nice if we have a ascending order for all the policy_specifier to be added. 
Revert it as it's not the general recognition.